editor/plugins/tiles/tile_set_atlas_source_editor.h

```diff
@@ -259,7 +259,9 @@ class TileSetAtlasSourceEditor : public HSplitContainer {
 	void _unhandled_key_input(const Ref<InputEvent> &p_event);
 
 	// -- Misc --
+	void _auto_create_tiles(Vector<Ref<TileSetAtlasSource>> const &new_tile_set_atlas_sources);
```

Comment:
Better naming?

Reply:
Should be `const Vector<...> &`, and the parameter should start with `p_`.